support-frontend/app/controllers/CreateSubscriptionController.scala

```diff
@@ -61,37 +61,43 @@ class CreateSubscriptionController(
 
   import actionRefiners._
 
+  // This is just for readability
+  private type CreateRequest = OptionalAuthRequest[CreateSupportWorkersRequest]
+
   def create: EssentialAction =
     LoggingAndAlarmOnFailure {
       MaybeAuthenticatedActionOnFormSubmission.async(circe.json[CreateSupportWorkersRequest]) { implicit request =>
         implicit val settings: AllSettings = settingsProvider.getAllSettings()
-        SafeLogger.info(s"${request.uuid}: debug info ${request.body.debugInfo}")
+
+        logDetailedMessage("attempting to create")
+        val json = request.body.copy(debugInfo = None).asJson
```

Comment:
are we stopping logging the debugInfo because it has already been logged in the raw incming request?

Reply:
Indeed. From the description:

Another noise reduction strategy is to remove the debug info from this log path and to only log the data creating the request. [This data is still logged in the `CustomActionBuilder`](https://github.com/guardian/support-frontend/blob/main/support-frontend/app/actions/CustomActionBuilders.scala#L85) (should have dropped it here as a comment).

We've found it makes debugging very hard. @GHaberis suggested we might have had this for when we were trying to reduce a large amount of alarms, where at the moment we only get a few and rarely look at this info.

Reply:
yeah this was hacked in because we had no clue how to reproduce bad requests where e.g. someone selected the payment method first, then went back and filled in the delivdry county, and then the billing address...etc
It was kind of a poor version of quantum metric (although we can include PII in it) to get the redux history in the logs.
If quantum metric covers everything we need then all the more reason to retire it.
I know there were some issues where state changes that went via react useState got missed in the debug info, so it may be less useful anyway.

Reply:
and yes self-comments are good 💯 , I tend to effectively self-review my PRs because then it keeps the explanation in context (and can help github archaeologists!)